### DISCUSSION TITLE
I got my token! ...Now what? (CLI)

### DISCUSSION THREAD

**Discussion by starringer** [Tyrrrz/DiscordChatExporter#1264] [Q&A]
I'm trying to use the CLI, as it can export an entire server where the GUI doesn't seem to be able to do so.

For the GUI, there's a very clear field to put the token into.

I'm certain there's something extremely obvious I'm missing, but how do I use the token in the CLI? All it tells me is how to get it, not what to do with it.

**Comment by Tyrrrz** [comment]
In GUI you can select multiple changes by using standard multi-select patterns, such as holding CTRL or SHIFT.

To use the token in CLI, pass it via `--token abcdef`. Make sure you're calling the right command. By your description, it looks like you're calling the `guide` command, which just explains how to use the program.
